geo/cube_small.dat

```diff
@@ -0,0 +1,107 @@
+0.000000000000000   0.000000000000000   1.000000000000000   1.000000000000000   
+0.000000000000000   0.000000000000000   0.000000000000000   0.000000000000000   
+0.000000000000000   0.000000000000000   0.250000000000000   0.250000000000000   
+0.000000000000000   0.250000000000000   0.000000000000000   0.250000000000000   
+1.000000000000000   1.000000000000000   1.000000000000000   1.000000000000000   
+PATCH 2 
+1 1 
+2 2 
+0.000000000000000   0.000000000000000   1.000000000000000   1.000000000000000   
+0.000000000000000   0.000000000000000   1.000000000000000   1.000000000000000   
+0.250000000000000   0.250000000000000   0.250000000000000   0.250000000000000   
+0.000000000000000   0.250000000000000   0.000000000000000   0.250000000000000   
+0.000000000000000   0.000000000000000   0.250000000000000   0.250000000000000   
+1.000000000000000   1.000000000000000   1.000000000000000   1.000000000000000   
+PATCH 3 
+1 1 
+2 2 
+0.000000000000000   0.000000000000000   1.000000000000000   1.000000000000000   
+0.000000000000000   0.000000000000000   1.000000000000000   1.000000000000000   
+0.000000000000000   0.250000000000000   0.000000000000000   0.250000000000000   
+0.000000000000000   0.000000000000000   0.000000000000000   0.000000000000000   
+0.000000000000000   0.000000000000000   0.250000000000000   0.250000000000000   
+1.000000000000000   1.000000000000000   1.000000000000000   1.000000000000000   
+PATCH 4 
+1 1 
+2 2 
+0.000000000000000   0.000000000000000   1.000000000000000   1.000000000000000   
+0.000000000000000   0.000000000000000   1.000000000000000   1.000000000000000   
+0.000000000000000   0.000000000000000   0.250000000000000   0.250000000000000   
+0.250000000000000   0.250000000000000   0.250000000000000   0.250000000000000   
+0.000000000000000   0.250000000000000   0.000000000000000   0.250000000000000   
+1.000000000000000   1.000000000000000   1.000000000000000   1.000000000000000   
+PATCH 5 
+1 1 
+2 2 
+0.000000000000000   0.000000000000000   1.000000000000000   1.000000000000000   
+0.000000000000000   0.000000000000000   1.000000000000000   1.000000000000000   
+0.000000000000000   0.000000000000000   0.250000000000000   0.250000000000000   
+0.000000000000000   0.250000000000000   0.000000000000000   0.250000000000000   
+0.000000000000000   0.000000000000000   0.000000000000000   0.000000000000000   
+1.000000000000000   1.000000000000000   1.000000000000000   1.000000000000000   
+PATCH 6 
+1 1 
+2 2 
+0.000000000000000   0.000000000000000   1.000000000000000   1.000000000000000   
+0.000000000000000   0.000000000000000   1.000000000000000   1.000000000000000   
+0.000000000000000   0.250000000000000   0.000000000000000   0.250000000000000   
+0.000000000000000   0.000000000000000   0.250000000000000   0.250000000000000   
+0.250000000000000   0.250000000000000   0.250000000000000   0.250000000000000   
+1.000000000000000   1.000000000000000   1.000000000000000   1.000000000000000   
+INTERFACE 1 
```

Comment:
(optional) we do not need the interfaces. For consistency we should remove them.